plugins/experimental/url_sig/url_sig.c

```diff
@@ -497,7 +504,7 @@ TSRemapDoRemap(void *ih, TSHttpTxn txnp, TSRemapRequestInfo *rri)
 
   signed_part[strlen(signed_part) - 1] = '?'; // chop off the last /, replace with '?'
```

Comment:
`signed_part` might not be nul-terminated now, if we were unlucky enough to exactly fill the buffer.